modules/ocf_decal/manifests/website.pp

```diff
@@ -0,0 +1,59 @@
+class ocf_decal::website {
+
+  file {
+    ['/srv/www', '/srv/www/decal']:
+      ensure  => directory,
+      owner   => ocfdecal,
+      group   => www-data,
+      require => User['ocfdecal'];
+    ['/srv/ssl', '/srv/ssl/decal']:
+      ensure  => directory,
+      owner   => ocfdecal,
+      group   => www-data,
+      mode    => '0440',
+      require => User['ocfdecal'];
+    '/srv/ssl/decal/decal.key':
+      source  => 'puppet:///private/decal.key',
+      mode    => '0440';
+    '/srv/ssl/decal/decal.crt':
+      source  => 'puppet:///private/decal.crt',
+      mode    => '0440';
+    '/etc/ssl/certs/lets-encrypt.crt':
+      source  => 'puppet:///private/lets-encrypt.crt',
+      mode    => '0640';
+  }
+
+  apache::vhost { 'decal-redirect':
+    servername => 'decal.ocf.berkeley.edu',
+    serveraliases => [
+      'decal',
+      'decal.ocf.io',
+      'decal.xcf.sh',
+      'decal.xcf.berkeley.edu'
+    ],
+    port => 80,
+    docroot => '/srv/www/decal',
+    redirect_status => 'permanent',
+    redirect_dest   => 'https://decal.ocf.berkeley.edu',
+  }
+
+  apache::vhost { 'decal-ssl':
+    servername => 'decal.ocf.berkeley.edu',
+    serveraliases => [
+      'decal',
+      'decal.ocf.io',
+      'decal.xcf.sh',
+      'decal.xcf.berkeley.edu',
```

Comment:
I still think you'd want to redirect these to have a more canonical domain instead of 5 different domains that people can use to get to the website. It's the standard pattern we follow elsewhere anyway.

Reply:
I believe you made these changes yourself, so that should be done/included in the latest commit.

Reply:
I don't think so, I didn't change anything related to this, the only changes I made were to enable `mod_rewrite`, fix the redirect from http (adding an ending slash), and enable overrides from `.htaccess` files. I still think this should be done though.

Reply:
(Can be done after merging this though, this one is not a big deal either way)